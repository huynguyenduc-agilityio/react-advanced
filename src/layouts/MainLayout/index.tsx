import { Outlet } from 'react-router-dom';

// Contexts
import { ConfirmationDialogProvider } from '@/contexts';

// Components
import { ConfirmDialog, Navbar } from '@/components';

const MainLayout = () => {
  return (
    <ConfirmationDialogProvider>
      <Navbar>
        <Outlet />
      </Navbar>
      <ConfirmDialog />
    </ConfirmationDialogProvider>
  );
};

export default MainLayout;
