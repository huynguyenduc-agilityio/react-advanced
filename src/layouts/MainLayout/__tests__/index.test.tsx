import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';

// Layout
import MainLayout from '@/layouts/MainLayout';

describe('MainLayout', () => {
  it('renders correctly and matches snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <MainLayout />
      </MemoryRouter>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
