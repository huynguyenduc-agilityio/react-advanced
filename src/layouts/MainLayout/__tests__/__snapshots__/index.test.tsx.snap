// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout renders correctly and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="css-ugqpym"
  >
    <nav
      class="css-1i9ywij"
    >
      <div
        class="chakra-stack css-iy87n6"
      >
        <div
          class="css-13rbvqk"
        >
          <div
            class="chakra-stack css-1g0vuyp"
          >
            <img
              alt="Logo"
              class="chakra-image css-nok893"
              src="[object Object]"
            />
            <h1
              class="chakra-text css-2tfj0r"
            >
              Dashdark X
            </h1>
          </div>
        </div>
        <svg
          class="chakra-icon css-lxef9e"
          fill="none"
          focusable="false"
          height="1em"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="2"
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="m9 7-5 5 5 5"
          />
          <path
            d="m15 7 5 5-5 5"
          />
        </svg>
      </div>
      <div
        class="css-mrftg7"
      >
        <div
          class="chakra-input__group css-29d2s5"
          data-group="true"
        >
          <input
            class="chakra-input css-1sf8ua8"
            placeholder="Search for..."
            type="text"
          />
          <div
            class="chakra-input__left-element css-1tldz2r"
          >
            <svg
              class="chakra-icon css-16qqcyx"
              fill="currentColor"
              focusable="false"
              height="1em"
              id="search"
              stroke="currentColor"
              stroke-width="0"
              version="1.1"
              viewBox="0 0 24 24"
              width="1em"
              x="0px"
              xmlns="http://www.w3.org/2000/svg"
              y="0px"
            >
              <g>
                <path
                  d="M20.031,20.79c0.46,0.46,1.17-0.25,0.71-0.7l-3.75-3.76c1.27-1.41,2.04-3.27,2.04-5.31
		c0-4.39-3.57-7.96-7.96-7.96s-7.96,3.57-7.96,7.96c0,4.39,3.57,7.96,7.96,7.96c1.98,0,3.81-0.73,5.21-1.94L20.031,20.79z
		 M4.11,11.02c0-3.84,3.13-6.96,6.96-6.96c3.84,0,6.96,3.12,6.96,6.96c0,3.84-3.12,6.96-6.96,6.96C7.24,17.98,4.11,14.86,4.11,11.02
		z"
                />
              </g>
            </svg>
          </div>
        </div>
      </div>
      <div
        class="chakra-accordion css-0"
      >
        <div
          class="chakra-stack css-1gmzflo"
        >
          <div
            class="css-0"
          >
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:r1:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="0"
                id="accordion-button-:r1:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      fill="none"
                      height="15"
                      viewBox="0 0 15 15"
                      width="15"
                    >
                      <path
                        d="M0.0683594 7.0523C0.0683594 6.63158 0.249846 6.23091 0.567115 5.95119L6.07891 1.09175C6.643 0.594418 7.49372 0.594418 8.05781 1.09175L13.5696 5.95119C13.8869 6.23091 14.0684 6.63158 14.0684 7.0523V13.2446C14.0684 14.0588 13.4021 14.7188 12.5802 14.7188H1.55656C0.73465 14.7188 0.0683594 14.0588 0.0683594 13.2446V7.0523Z"
                        fill="#AEB9E1"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Dashboard
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:r1:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:r1:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  >
                    <a
                      href="/*"
                      style="width: 100%;"
                    >
                      <div
                        class="css-1f0q9tr"
                        role="group"
                      >
                        All pages
                      </div>
                    </a>
                    <a
                      href="/report"
                      style="width: 100%;"
                    >
                      <div
                        class="css-h1kcc"
                        role="group"
                      >
                        Report
                      </div>
                    </a>
                    <a
                      href="/*"
                      style="width: 100%;"
                    >
                      <div
                        class="css-1f0q9tr"
                        role="group"
                      >
                        Products
                      </div>
                    </a>
                    <a
                      href="/*"
                      style="width: 100%;"
                    >
                      <div
                        class="css-1f0q9tr"
                        role="group"
                      >
                        Task
                      </div>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-0"
          >
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:r3:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="1"
                id="accordion-button-:r3:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      class="chakra-icon css-13otjrl"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 576 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M259.3 17.8L194 150.2 47.9 171.5c-26.2 3.8-36.7 36.1-17.7 54.6l105.7 103-25 145.5c-4.5 26.3 23.2 46 46.4 33.7L288 439.6l130.7 68.7c23.2 12.2 50.9-7.4 46.4-33.7l-25-145.5 105.7-103c19-18.5 8.5-50.8-17.7-54.6L382 150.2 316.7 17.8c-11.7-23.6-45.6-23.9-57.4 0z"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Features
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:r3:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:r3:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-0"
          >
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:r5:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="2"
                id="accordion-button-:r5:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      class="chakra-icon css-13otjrl"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Users
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:r5:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:r5:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  >
                    <a
                      href="/users/add-user"
                      style="width: 100%;"
                    >
                      <div
                        class="css-1f0q9tr"
                        role="group"
                      >
                        Add Users
                      </div>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-0"
          >
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:r7:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="3"
                id="accordion-button-:r7:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      fill="none"
                      height="15"
                      viewBox="0 0 10 15"
                      width="10"
                    >
                      <path
                        clip-rule="evenodd"
                        d="M5.00357 0.5C5.51784 0.5 5.93473 0.916896 5.93473 1.43116V1.90222C6.36312 1.90237 6.7873 1.98682 7.18308 2.15076C7.57923 2.31485 7.93917 2.55535 8.24236 2.85855C8.54556 3.16174 8.78607 3.52169 8.95016 3.91783C9.11425 4.31398 9.1987 4.73856 9.1987 5.16734C9.1987 5.68161 8.7818 6.09851 8.26754 6.09851C7.75327 6.09851 7.33637 5.68161 7.33638 5.16734C7.33638 4.98312 7.30009 4.80071 7.22959 4.63051C7.1591 4.46032 7.05577 4.30567 6.9255 4.17541C6.79524 4.04515 6.6406 3.94182 6.4704 3.87132C6.3002 3.80082 6.11779 3.76454 5.93357 3.76454H3.833C3.46095 3.76454 3.10414 3.91233 2.84107 4.17541C2.57799 4.43849 2.43019 4.7953 2.43019 5.16734C2.43019 5.53939 2.57799 5.8962 2.84107 6.15928C3.10414 6.42235 3.46095 6.57015 3.833 6.57015H6.40036C7.26633 6.57015 8.09683 6.91415 8.70916 7.52648C9.32149 8.13881 9.66549 8.96931 9.66549 9.83528C9.66549 10.7012 9.32149 11.5317 8.70916 12.1441C8.09683 12.7564 7.26633 13.1004 6.40036 13.1004H5.93473V13.5688C5.93473 14.0831 5.51784 14.5 5.00357 14.5C4.48931 14.5 4.07241 14.0831 4.07241 13.5688V13.1004H3.5996C2.73364 13.1004 1.90314 12.7564 1.29081 12.1441C0.678476 11.5317 0.334473 10.7012 0.334473 9.83528C0.334473 9.32101 0.751368 8.90412 1.26563 8.90412C1.7799 8.90412 2.1968 9.32101 2.1968 9.83528C2.1968 10.2073 2.34459 10.5641 2.60767 10.8272C2.87075 11.0903 3.22756 11.2381 3.5996 11.2381H4.96485C4.97769 11.2376 4.9906 11.2373 5.00357 11.2373C5.01654 11.2373 5.02945 11.2376 5.0423 11.2381H6.40036C6.77241 11.2381 7.12922 11.0903 7.3923 10.8272C7.65537 10.5641 7.80317 10.2073 7.80317 9.83528C7.80317 9.46323 7.65537 9.10642 7.3923 8.84335C7.12922 8.58027 6.77241 8.43247 6.40036 8.43247H3.833C2.96703 8.43247 2.13653 8.08847 1.5242 7.47614C0.911873 6.86381 0.567869 6.03331 0.567869 5.16734C0.567869 4.30138 0.911873 3.47088 1.5242 2.85855C2.13653 2.24622 2.96703 1.90221 3.833 1.90221H4.07241V1.43116C4.07241 0.916896 4.48931 0.5 5.00357 0.5Z"
                        fill="#AEB9E1"
                        fill-rule="evenodd"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Pricing
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:r7:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:r7:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-0"
          >
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:r9:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="4"
                id="accordion-button-:r9:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      fill="none"
                      height="14"
                      viewBox="0 0 14 14"
                      width="14"
                    >
                      <path
                        d="M3.36257 13.72C3.21408 13.72 3.07168 13.661 2.96668 13.556C2.86169 13.451 2.8027 13.3086 2.8027 13.1601V10.1718C2.49555 10.3184 2.15515 10.3811 1.81594 10.3538C1.52872 10.3327 1.24969 10.2485 0.998699 10.1073C0.747707 9.96611 0.530926 9.77132 0.363785 9.53679C0.196643 9.30227 0.0832509 9.03378 0.0316773 8.75045C-0.0198963 8.46712 -0.00838304 8.1759 0.0653976 7.89752C0.139178 7.61914 0.273412 7.36045 0.458547 7.13985C0.643681 6.91925 0.875164 6.74217 1.13652 6.62122C1.39788 6.50026 1.68268 6.4384 1.97067 6.44004C2.25865 6.44168 2.54274 6.50677 2.8027 6.63069V3.64241C2.8027 3.49393 2.86169 3.35152 2.96668 3.24653C3.07168 3.14153 3.21408 3.08255 3.36257 3.08255H6.63078C6.48426 2.7754 6.42149 2.43499 6.44882 2.09579C6.46992 1.80857 6.55408 1.52954 6.69529 1.27855C6.8365 1.02755 7.0313 0.810772 7.26582 0.643631C7.50035 0.47649 7.76883 0.363097 8.05217 0.311524C8.3355 0.25995 8.62672 0.271463 8.9051 0.345244C9.18347 0.419024 9.44217 0.553259 9.66276 0.738393C9.88336 0.923527 10.0604 1.15501 10.1814 1.41637C10.3024 1.67772 10.3642 1.96253 10.3626 2.25051C10.3609 2.5385 10.2958 2.82258 10.1719 3.08255H13.4401C13.5886 3.08255 13.731 3.14153 13.836 3.24653C13.941 3.35152 14 3.49393 14 3.64241V6.63069C13.6929 6.48417 13.3524 6.4214 13.0132 6.44874C12.726 6.46984 12.447 6.55399 12.196 6.6952C11.945 6.83641 11.7282 7.03121 11.5611 7.26574C11.3939 7.50026 11.2805 7.76874 11.229 8.05208C11.1774 8.33541 11.1889 8.62663 11.2627 8.90501C11.3365 9.18339 11.4707 9.44208 11.6558 9.66268C11.841 9.88328 12.0725 10.0604 12.3338 10.1813C12.5952 10.3023 12.88 10.3641 13.168 10.3625C13.456 10.3609 13.74 10.2958 14 10.1718V13.1601C14 13.3086 13.941 13.451 13.836 13.556C13.731 13.661 13.5886 13.72 13.4401 13.72H3.36257Z"
                        fill="#AEB9E1"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Integrations
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:r9:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:r9:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-0"
          >
            <hr
              aria-orientation="horizontal"
              class="chakra-divider css-r6x8k1"
            />
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:rb:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="5"
                id="accordion-button-:rb:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      class="chakra-icon css-13otjrl"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 24 24"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M0 0h24v24H0V0z"
                        fill="none"
                      />
                      <path
                        d="M19.14 12.94c.04-.3.06-.61.06-.94 0-.32-.02-.64-.07-.94l2.03-1.58a.49.49 0 0 0 .12-.61l-1.92-3.32a.488.488 0 0 0-.59-.22l-2.39.96c-.5-.38-1.03-.7-1.62-.94l-.36-2.54a.484.484 0 0 0-.48-.41h-3.84c-.24 0-.43.17-.47.41l-.36 2.54c-.59.24-1.13.57-1.62.94l-2.39-.96c-.22-.08-.47 0-.59.22L2.74 8.87c-.12.21-.08.47.12.61l2.03 1.58c-.05.3-.09.63-.09.94s.02.64.07.94l-2.03 1.58a.49.49 0 0 0-.12.61l1.92 3.32c.12.22.37.29.59.22l2.39-.96c.5.38 1.03.7 1.62.94l.36 2.54c.05.24.24.41.48.41h3.84c.24 0 .44-.17.47-.41l.36-2.54c.59-.24 1.13-.56 1.62-.94l2.39.96c.22.08.47 0 .59-.22l1.92-3.32c.12-.22.07-.47-.12-.61l-2.01-1.58zM12 15.6c-1.98 0-3.6-1.62-3.6-3.6s1.62-3.6 3.6-3.6 3.6 1.62 3.6 3.6-1.62 3.6-3.6 3.6z"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Settings
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:rb:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:rb:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  />
                </div>
              </div>
            </div>
          </div>
          <div
            class="css-0"
          >
            <div
              class="chakra-accordion__item css-xc62zp"
            >
              <button
                aria-controls="accordion-panel-:rd:"
                aria-expanded="false"
                class="chakra-accordion__button css-14qpop4"
                data-index="6"
                id="accordion-button-:rd:"
                type="button"
              >
                <div
                  class="css-1c88k4t"
                >
                  <div
                    class="css-540lu8"
                  >
                    <svg
                      fill="none"
                      height="15"
                      viewBox="0 0 14 15"
                      width="14"
                    >
                      <path
                        d="M10.2989 5.48867C10.2989 5.48867 9.17241 9.0289 9.09196 9.35074C9.09196 9.0289 8.2069 2.67258 8.2069 2.67258C6.27586 2.67258 5.22989 4.04039 4.74713 5.48867C4.74713 5.48867 3.43604 8.2462 3.35558 8.56804C3.27512 8.32666 3.05747 5.56913 3.05747 5.56913C2.89655 3.79901 1.28736 2.67258 0 2.67258L1.6092 12.3277C3.62069 12.3277 4.74713 10.9599 5.31034 9.51166C5.31034 9.51166 6.51724 6.37373 6.5977 6.21281C6.5977 6.37373 7.48276 12.3277 7.48276 12.3277C9.49425 12.3277 10.6207 11.0404 11.1839 9.67258L14 2.67258C11.908 2.67258 10.8621 4.04039 10.2989 5.48867Z"
                        fill="#AEB9E1"
                      />
                    </svg>
                  </div>
                  <div
                    class="css-195y58u"
                  >
                    Template pages
                  </div>
                  <svg
                    aria-hidden="true"
                    class="chakra-icon css-9ls6jt"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 20 20"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
              </button>
              <div
                class="chakra-collapse"
                style="overflow: hidden; display: none; opacity: 0; height: 0px;"
              >
                <div
                  aria-labelledby="accordion-button-:rd:"
                  class="chakra-accordion__panel css-1pom12u"
                  id="accordion-panel-:rd:"
                  role="region"
                >
                  <div
                    class="chakra-stack css-m3jj3s"
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="css-q8oktc"
      >
        <div
          class="css-j2at52"
        >
          <span
            class="chakra-avatar css-1ruhx4w"
          >
            <div
              aria-label="John Carter"
              class="chakra-avatar__initials css-0"
              role="img"
            >
              JC
            </div>
          </span>
          <div
            class="css-j7qwjs"
          >
            <p
              class="chakra-text css-19d3k7d"
            >
              John Carter
            </p>
            <p
              class="chakra-text css-u6p1bq"
            >
              Account settings
            </p>
          </div>
        </div>
        <svg
          aria-hidden="true"
          class="chakra-icon css-9ls6jt"
          fill="currentColor"
          focusable="false"
          height="1em"
          stroke="currentColor"
          stroke-width="0"
          viewBox="0 0 20 20"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            clip-rule="evenodd"
            d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
            fill-rule="evenodd"
          />
        </svg>
      </div>
      <div
        class="css-13z0ivi"
      >
        <button
          class="chakra-button css-nkp48u"
          type="button"
        >
          Get template
          <span
            class="chakra-button__icon css-1hzyiq5"
          >
            <svg
              aria-hidden="true"
              fill="none"
              focusable="false"
              height="1em"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="1em"
              xmlns="http://www.w3.org/2000/svg"
            >
              <line
                x1="5"
                x2="19"
                y1="12"
                y2="12"
              />
              <polyline
                points="12 5 19 12 12 19"
              />
            </svg>
          </span>
        </button>
      </div>
    </nav>
    <main
      class="css-1kxshok"
    />
  </div>
</DocumentFragment>
`;
